runtime/doc/builtin.txt

```diff
@@ -6947,10 +6947,10 @@ matchadd({group}, {pattern} [, {priority} [, {id} [, {dict}]]])
 matchaddpos({group}, {pos} [, {priority} [, {id} [, {dict}]]])
 		Same as |matchadd()|, but requires a list of positions {pos}
 		instead of a pattern. This command is faster than |matchadd()|
-		because it does not require to handle regular expressions and
-		sets buffer line boundaries to redraw screen. It is supposed
-		to be used when fast match additions and deletions are
-		required, for example to highlight matching parentheses.
+		because it does not handle regular expressions and it sets
```

Comment:
I would change `it sets` to just `sets` as before

Reply:
Without adding "it" `sets buffer line boundaries to redraw screen.` doesn't have a subject.

Reply:
I might be reading original wrong, for me first "it" is a subject: it doesn't handle ... and sets ...